extensions/amp-access/0.1/signin.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isArray} from '../../../src/types';
+import {isExperimentOn} from '../../../src/experiments';
+import {user} from '../../../src/log';
+
+
+/** @const */
+const TAG = 'amp-access-signin';
```

Comment:
I thought the latest was to call these `TAG`. No? FWIW, it's used as a tag as well in log calls.